### PR TITLE
data: RashidatJay

### PR DESCRIPTION
**Pull request by RashidatJay**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3726"></a>

### CODE REVIEW

File: data/RashidatJay.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Rashidatjay",
+      "icon": "FaTwitter"
+    },
+     "name": "Follow me on LinkedIn"
+     "url": "https://www.linkedin.com/in/rashidatjimoh/",
+     "icon": "FaLinkedIn"
+     {
```
Comment (ChinmayMhatre) [inline data/RashidatJay.json:16]: Please move this bracket before line 13 i.e before the name property

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: data/RashidatJay.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Rashidat Jimoh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student|| Technical Writer|| Developer",
+  "avatar": "https://github.com/RashidatJay.png",
```
Comment (devvsakib) [inline data/RashidatJay.json:6]: you can add tags here like this
image

**Review by devvsakib — CHANGES_REQUESTED** [review]

File: data/RashidatJay.json (hunk 2 above)
Comment (RashidatJay) [reply data/RashidatJay.json:7]: I have done this

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/RashidatJay.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Rashidat Jimoh",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/RashidatJay.json:3]: Following an improvement this is no longer required
```suggestion
```

File: data/RashidatJay.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  "type": "personal",
+  "bio": "Student|| Technical Writer|| Developer",
+  "tags": ["JavaScript", "python", "open source"],
+  "avatar": "https://github.com/RashidatJay.png",
```
Comment (SaraJaoude) [inline data/RashidatJay.json:7]: The avatar is no longer required
```suggestion
```

**Pull request merged**